src/runtime/mod.rs

```diff
@@ -983,8 +979,8 @@ mod test {
 
         let log_tempdir = tempfile::tempdir().expect("Unable to create tempdir");
         let cache_tempdir = tempfile::tempdir().expect("new cache temp dir");
-
-        mc.build_registry(&cache, cache_tempdir.path(), log_tempdir.path())
+        let http_conc = None;
```

Comment:
Nit: You should be able to skip this and just pass `&None` on the next line.

Reply:
done